### ISSUE TITLE
Could not find a declaration file for module 'react-treeselectjs

### ISSUE THREAD

**Issue by indvinoth** [dipson88/react-treeselectjs#12]
Thanks for creating the react wrapper but when I add the library to my Typescript project I'm getting the following error.

```
Could not find a declaration file for module 'react-treeselectjs'.
'project_path/node_modules/react-treeselectjs/dist/react-treeselectjs.mjs' implicitly has an 'any' type.
There are types at 'project_path/node_modules/react-treeselectjs/dist/react-treeselectjs.d.ts',
but this result could not be resolved when respecting package.json "exports". 
The 'react-treeselectjs' library may need to update its package.json or typings.
```

I managed to fix the error by updating the path in `tsconfig.json`

```
{
    "compilerOptions": {
        "paths": {
            "react-treeselectjs": ["./node_modules/react-treeselectjs/dist/react-treeselectjs.d.ts"]
        }
    },
}

```

If  possible please update `package.json` with proper path.

**Comment by dipson88** [comment]
@indvinoth Hi and thank you!
Hmm, I need to check it, and then I let you know the results.

**Comment by dipson88** [comment]
Was fixed in v0.4.0
